### PR TITLE
Reformatting Pages + Friend Feature

### PR DESCRIPTION
**Pull request by progammer**
- Reformated Pages and Side Bar for future navigation
- In blog/views.py, removed def home view function that was rendering blog/home.html (PostListView was already rendering blog/home.html)
- In blog/views.py, added def get_content_data inside PostListView to setup the context for blog/home.html
- In django_project/urls.py, added path("user/", include("users.urls"))
- Friend Request/Accept Features

### CODE REVIEW

File: silky-squirrels/.vscode/settings.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
 	"editor.formatOnSave": true,
-	"python.formatting.provider": "black"
+	"python.formatting.provider": "black",
+	"python.pythonPath": "C:\\Users\\caleb\\.virtualenvs\\silky-squirrels-Q9e-qDbz\\Scripts\\python.exe"
```
Comment (kmui2) [inline silky-squirrels/.vscode/settings.json]: Remove `"python.pythonPath"` here. I accidentally made you add this.

File: silky-squirrels/blog/views.py [hunk 2]
```diff
@@ -17,6 +13,14 @@ class PostListView(ListView):
     ordering = ["-date_posted"]
     paginate_by = 5
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        profile = Profile.objects.get(user=self.request.user)
+        context["posts"] = Post.objects.all()
+        context["allusers"] = User.objects.exclude(username=self.request.user)
```
Comment (kmui2) [inline silky-squirrels/blog/views.py]: I suggest excluding by `id` here:

```diff
- context["allusers"] = User.objects.exclude(username=self.request.user)
+ context["allusers"] = User.objects.exclude(id=self.request.user.id)
```

Comment (progammer) [reply silky-squirrels/blog/views.py]: Faster lookup?

Comment (kmui2) [reply silky-squirrels/blog/views.py]: yes

**Review by kmui2 — APPROVED** [review]

**Pull request merged**